### ISSUE TITLE
Central test area datastore repository

### ISSUE THREAD

**Issue by gregorbj** [gregorbj/Archive-VisionEval#121]
We need an efficient way to store and access a test area datastore. A test area datastore is essential to efficient development and testing of modules since most modules need to use data that is in a datastore. Currently I've been including two datastores in the 'test' directory of each package. In an 'initial_datastore' directory is a datastore with initial conditions. In the main directory is a datastore with the final conditions (after all package modules are run). The final datastore is then copied to the 'initial_datastore' folder for the next package being worked on. This results in a lot of duplicated data. It would take less space and perhaps be more efficient and less error-prone to read in the datastore.h5 file from the 'VERSPM' directory. This should be updated with the latest successful run of VERSPM. Might be good to have some functions to facilitate this and using the datastore in module testing.

**Comment by gregorbj** [comment]
This would require keeping a copy of the 'datastore.h5' file in the directory which we don't do now. It might also be good to keep a copy of the 'ModelState.Rda' file. This file makes it easier to read from the datastore. Having the test datastore would also be good for testing output and visualization functions and scripts.

**Comment by bstabler** [comment]
Once VERPAT and VERSPM are complete, the output datastore would be a great resource for developing new modules.

**Comment by bstabler** [comment]
this might be a good task for the pooled funds

**Comment by gregorbj** [comment]
I created a centralized datastore for VE-State test data in the sources/modules/Test_Data directory. The main reason for doing this now is that there is a lot of redundancy in storing test datastores in each of the module packages. While this isn't a problem for small test areas, it becomes a big problem for a large model area. I've altered the module test code in the VE-State modules so that it reads from and writes to this centralized datastore. It is a simple thing to do. This also advances the project towards the goal of creating centralized datastores that can be used for new module development and for regression testing. It would be simple to convert VE-RSPM and VE-RPAT datastores to centralized repositories as well since only some small changes to the test scripts are needed. 

It would be a good idea to keep copies of all the def and input files for the test dataset in the same area to ensure replicability.
